canonical-web-and-design/jaas-dashboard
# Unmount slide panels after animation completion

## Done

Unmount slide panels after animation completion

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify panel component is unmounted after animation completes
